scripts/mutect2_wdl/mutect2_multi_sample.wdl

```diff
@@ -57,7 +58,8 @@ workflow Mutect2 {
 				dbsnp_index=dbsnp_index,
 				cosmic=cosmic,
 				cosmic_index=cosmic_index,
-				picard_jar=picard_jar
+				picard_jar=picard_jar,
```

Comment:
We use `MergeVCFs` and `IntervalListTools` to scatter over intervals -- remember, this is calling the single-sample M2 (sub)-*workflow*, not the M2 *task*.